roles/openshift_node_group/tasks/create_config.yml

```diff
@@ -42,7 +42,6 @@
       src: "{{ mktempout.stdout }}/node-config.yaml"
       edits: "{{ openshift_node_group_edits | union(openshift_node_labels_edit) }}"
```

Comment:
openshift_node_group_edits is empty, and set in vars?  We should just remove this variable then.

Reply:
I'm fine with that, but we can tackle that later.